packages/material-ui-system/src/breakpoints.js

```diff
@@ -85,4 +86,37 @@ function breakpoints(styleFunction) {
   return newStyleFunction;
 }
 
+function createEmptyBreakpointObject(breakpointsInput) {
```

Comment:
These methods are extracted here, so they can be reused both in the `styleFunction` as well as the `styleFunctionSx` utilities.